Pod/Classes/Deselection/UIViewController+Deselection.swift

```diff
@@ -38,27 +44,59 @@ public extension UIViewController {
     ///  controller's `viewWillAppear(_:)` method.
     ///
     ///  - parameter tableView: The table view in which to perform deselection/reselection.
-    func rz_smoothlyDeselectRows(tableView tableView: UITableView?) {
+    func rz_smoothlyDeselectRows(tableView: UITableView?) {
```

Comment:
I think `rz_smoothlyDeselectRows(in tableView: UITableView?)` would be a clearer call site with 3.0 naming conventions. Not sure if 3.0 conventions are important at this point.